docs/src/guide/tutorial.jl

```diff
@@ -50,12 +50,18 @@ scfres = self_consistent_field(basis, tol=1e-8);
 # is not yet converged to 8 digits.
 #
 # That's it! Now you can get various quantities from the result of the SCF.
-# For instance, the energies:
+# For instance, the different components of the energy:
 scfres.energies
-# Eigenvalues: here `scfres.eigenvalues` returns a 7x8 Array where 7 is the
-# number of eigenvalues that are computed and 8 the number of kpoints.
-hcat(scfres.eigenvalues...)
-# Use `scfres.occupation` to see the number of electrons on each energy level.
+
+# Eigenvalues: 
+hcat(scfres.eigenvalues...) # eigenvalues is an array of arrays; this converts it to a matrix
```

Comment:
That's too long a comment, makes it hard to process for new users. Include as explanatory text above.